lib/kindle_highlights/client.rb

```diff
@@ -85,7 +85,7 @@ def mechanize_agent
 
     def initialize_mechanize_agent
       mechanize_agent                        = Mechanize.new
-      mechanize_agent.user_agent_alias       = 'Windows Mozilla'
+      mechanize_agent.user_agent_alias       = Mechanize::AGENT_ALIASES.keys.grep(/\A(Linux|Mac|Windows)/).sample
```

Comment:
The reason to only use these from the [full contents of the constant](https://github.com/sparklemotion/mechanize/blob/master/lib/mechanize.rb#L115) is to avoid getting a mobile view which may or may not be different.